scripts/mkstatic.py

```diff
@@ -504,6 +504,7 @@ def __init__(self, **kwargs):
         self._sdf_name = None
         self._all_schemes_called = None
         self._all_subroutines_called = None
+        self._call_tree = None
```

Comment:
The `_call_tree` property changes type, which could be problematic. It becomes a list in `make_call_tree`.